Receipt/Sources/ListReceiptController.swift

```diff
@@ -39,13 +39,16 @@ public final class ListReceiptController: UITableViewController {
         title = "title_receipts".localized()
         largeTitle()
         setViewBackgroundColor()
-
         navigationItem.backBarButtonItem = UIBarButtonItem.back
+        initializePresenter()
+        presenter.listReceipts()
+        setupListReceiptTableView()
+    }
+
+    private func initializePresenter() {
```

Comment:
I am not sure but would it make sense to introduce this method in `UITableViewController` at default access?